p22/benches/benchmark_tests.rs

```diff
@@ -0,0 +1,29 @@
+#![feature(test)]
+extern crate test;
+use p22::calc::{fibonacci_loop, fibonacci_rec};
+use std::hint::black_box;
+use test::Bencher;
+#[bench]
+fn benchmark_fibonacci_loop(b: &mut Bencher) {
+    b.iter(|| black_box(fibonacci_loop(6)));
+}
+#[bench]
+fn benchmark_fibonacci_loop2(b: &mut Bencher) {
+    b.iter(|| {
+        for i in 0..10 {
+            black_box(fibonacci_loop(i));
+        }
```

Comment:
Usually it's better to benchmark function call with a fixed parameter, especially in cases when performance strongly depends on input parameters. It's better to write several benchmark functions, e.g. `fibonacci_loop_10`, `fibonacci_loop_20`, etc.